scss/bitstyles/atoms/card/settings.scss

```diff
@@ -4,3 +4,4 @@ $bitstyles-card-box-shadow:
   0 spacing('xs') spacing('m') rgba(palette('brand-1', '100'), 0.1),
   0 spacing('xxxs') spacing('xxs') rgba(palette('gray', '90'), 0.04),
   0 0 spacing('xxs') rgba(palette('gray', '90'), 0.1) !default;
+$bitstyles-card-breakpoints: ('base', 'm', 'l') !default;
```

Comment:
So if I get this right, `base` always needs to be there otherwise we have no `a-card` at all, rigth? Would it maybe make  sense to use `list.join` to add `base` (not sure if this is valid Sass 🙈 )
```
@each $breakpoint-alias in list.join('base', $bitstyles-card-breakpoints) {
  @if $breakpoint-alias == 'base' {
    @include output-block(
      $object-level: 'a',
      $classname-root: 'card'
    ) {
      @include card;
    }
  }

  @else {
    @include media-query($breakpoint-alias) {
      @include output-block(
        $object-level: 'a',
        $classname-root: 'card',
        $breakpoint-suffix: '\\\@#{$breakpoint-alias}'
      ) {
        @include card;
      }
    }
  }
}
```

Reply:
Yes that would totally work (it’s correct Sass too 🎉:D). But maybe it’s simpler to just output the `base` version outside the loop, wdyt?

```scss
@include output-block(
  $object-level: 'a',
  $classname-root: 'card'
) {
  @include card;
}

@each $breakpoint-alias in $bitstyles-card-breakpoints {
  @include media-query($breakpoint-alias) {
    @include output-block(
      $object-level: 'a',
      $classname-root: 'card',
      $breakpoint-suffix: '\\\@#{$breakpoint-alias}'
    ) {
      @include card;
    }
  }
}
```

The intent to output base then loop over breakpoints seems clearer, and it gets rid of the `if` statement so the rest of the loop is more readable too, I think

Reply:
Yes, that is even better 💯

Reply:
Perfect, done! 👍